lib/Conversion/HandshakeToFIRRTL/HandshakeToFIRRTL.cpp

```diff
@@ -254,14 +265,22 @@ static Value createConstantOp(FIRRTLType opType, APInt value,
   return Value();
 }
 
-static Type getHandshakeBundleDataType(BundleType bundle) {
-  if (auto dataType = bundle.getElementType("data")) {
-    auto intType = dataType.cast<firrtl::IntType>();
-    return IntegerType::get(bundle.getContext(), intType.getWidthOrSentinel(),
+static Type getBundleElementType(Type type) {
```

Comment:
While it wasn't there before, a comment to preface this function would be a nice addition.